webpack/assets/javascripts/react_app/components/CPUCoresInput/__tests__/__snapshots__/CPUCoresInput.test.js.snap

```diff
@@ -22,6 +22,7 @@ exports[`CPUCoresInput rendering should render with default props 1`] = `
           className=""
           disabled={false}
           format={null}
+          id=""
```

Comment:
Why `""` is a default, won't it cause an empty id attribute? :thinking:

Reply:
@ezr-ondrej What will be good in your opinion as a default id?

Reply:
I'd say `null`, but I'm not sure and didn't tested what that results in, so it's not big deal if you leave an empty string :)